riotfile.py

```diff
@@ -2176,5 +2176,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="futures",
+            command="pytest {cmdargs} tests/contrib/futures",
+            venvs=[
+                Venv(pys=["2.7"], pkgs={"futures": ["~=3.0", "~=3.1", "~=3.2", "~=3.4"]}),
+                Venv(
+                    pys=select_pys(min_version="3.5"),
```

Comment:
we don't need futures here because it is built-in, only need the pkgs above since they are a backport for Python 2.7?

(maybe a comment would be a nice to have)

Reply:
correct, sure i can add a comment